KaanMol/AGV-1.2
# Feature/bluetooth

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #16

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Please provide the name of the unit test that has been created/modified. 
Please also include the steps that the test takes to verify the test is reliable and works.

**Test A**
1. Make a connection via Putty
2. Press the wasd keys slowly to see if the robot will move

# Checklist:
